agents/utils/configs.py

```
@@ -0,0 +1,90 @@
+
+        agents: list[AgentConfig] = []
+        for tokenId in range(1, token_counter):
+            address = self.contract.functions.agentWallets(tokenId).call()
+            agent_contract = self.w3.eth.contract(address=address, abi=self.agent_abi)
+            
+            backstory = agent_contract.functions.backstory.call()
+            coin_base_wallet_id = agent_contract.functions.coin_base_wallet_id.call()
+            name = agent_contract.functions.name.call()
+
+            # Fetch wallets
+            wallet = Wallet.fetch(address)
```

Ruff (0.8.2)

Local variable `wallet` is assigned to but never used

Remove assignment to unused variable `wallet`

(F841)